asusd/src/lib.rs

```diff
@@ -130,7 +132,7 @@ pub fn print_board_info() {
 }
 
 pub trait Reloadable {
-    fn reload(&mut self) -> impl std::future::Future<Output = Result<(), RogError>> + Send;
```

Comment:
Please avoid these sorts of changes if they are not directly related to the task on hand.